quick-marc/src/main/resources/domain/mod-quick-marc/features/quick-marc-record-status.feature

```diff
@@ -7,32 +7,9 @@ Feature: Test quickMARC for record status
     * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json'  }
     * def headersUserOctetStream = { 'Content-Type': 'application/octet-stream', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json'  }
 
-    * def result = callonce read('classpath:domain/mod-quick-marc/features/setup/setup.feature')
-    * def testInstanceId = result.response.sourceRecords[0].externalIdsHolder.instanceId
-
 #   ================= positive test cases =================
   Scenario: Retrieve record status after record creation
```

Comment:
I think we also should have a positive case for check a response from POST endpoint

Reply:
Added positive cases and checking qmRecordId, jobExecutionId on uuid